sdk/pinocchio/src/instruction.rs

```diff
@@ -135,25 +135,25 @@ impl<'a> AccountMeta<'a> {
 
     /// Creates a new readonly `AccountMeta`.
     #[inline(always)]
-    pub fn readonly(pubkey: &'a Pubkey) -> Self {
+    pub const fn readonly(pubkey: &'a Pubkey) -> Self {
```

Comment:
Not sure if these `const` qualifications are related to the other work you are doing in this PR.
If they are not, you might be able to get them in a separate PR faster.